### PR TITLE
Help texts for main windows

### PR DESCRIPTION
**Pull request by jhkl**
Fixes issue #456

### CODE REVIEW

File: src/ingameWindows/iwMerchandiseStatistics.cpp [hunk 1]
```diff
@@ -110,6 +112,12 @@ void iwMerchandiseStatistics::Msg_ButtonClick(const unsigned int ctrl_id)
     {
         case 16: // Hilfe
         {
+            WINDOWMANAGER.Show(new iwHelp(GUI_ID(CGI_HELP),
+                _("The merchandise statistics window allows you to check the quantities "
+                  "of your merchandise.By clicking the left mouse button you can switch "
```
Comment (Flow86) [inline src/ingameWindows/iwMerchandiseStatistics.cpp]: here's a space missing after the point

File: src/ingameWindows/iwMilitary.cpp [hunk 2]
```diff
@@ -110,6 +112,21 @@ void iwMilitary::Msg_ButtonClick(const unsigned ctrl_id)
...
+        case 20:
+        {
+            WINDOWMANAGER.Show(new iwHelp(GUI_ID(CGI_HELP), _(
+                "This is where you can make adjustments to all military matters. "
+                "The upper value corresponds to the recruiting rate of your army. "
+                "The higher it is, the more inhabitants are recruited as soldiers. "
+                "Below this is the setting to protect your huts.If this value is "
+                "set at maximum, your huts are defended by the strongest unit. To "
+                "raise the number of attackers leaving your huts per attack, choose "
+                "the next setting.The number of defenders who counter the enemy in "
```
Comment (Flow86) [inline src/ingameWindows/iwMilitary.cpp]: here too

File: src/ingameWindows/iwShip.cpp [hunk 3]
```diff
@@ -123,6 +125,15 @@ void iwShip::Msg_PaintAfter()
 
 void iwShip::Msg_ButtonClick(const unsigned int ctrl_id)
 {
+    if (ctrl_id == 2) // Hilfe
+    {
+        WINDOWMANAGER.Show(new iwHelp(GUI_ID(CGI_HELP),
+            _("The ship register contains all the ships in your fleet.Here you can monitor "
```
Comment (Flow86) [inline src/ingameWindows/iwShip.cpp]: and here

File: src/ingameWindows/iwStatistics.cpp [hunk 4]
```diff
@@ -168,7 +170,11 @@ void iwStatistics::Msg_ButtonClick(const unsigned int ctrl_id)
 
         case 25: // Hilfe
         {
-            // TODO Help!
+            WINDOWMANAGER.Show(new iwHelp(GUI_ID(CGI_HELP), _(
+                "This window allows a direct comparison with the enemies. "
+                "Factors such as the wealth, territorial area, inhabitants, "
+                "etc.of all parties can be compared.This data can be shown "
```
Comment (Flow86) [inline src/ingameWindows/iwStatistics.cpp]: and here

**Comment by Flow86** [comment]
we should perhaps create one file with all help texts, like the BUILDING_NAMES "variable"

**Comment by jhkl** [comment]
Thought about that as well (actually started with that), but: What is the benefit? 

The building help strings are in a extra file, because all buildings are one class.

The windows do not share anything (except being IngameWindows). If you modify a window, you probably want to modify the help text. Changing both independently does not really makes sense.

Will fix the spaces. Somehow they got lost in copy & pasting.

File: src/ingameWindows/iwShip.cpp [hunk 5]
```diff
@@ -129,7 +129,7 @@ void iwShip::Msg_ButtonClick(const unsigned int ctrl_id)
     {
         WINDOWMANAGER.Show(new iwHelp(GUI_ID(CGI_HELP),
             _("The ship register contains all the ships in your fleet.Here you can monitor "
```
Comment (Flamefire) [inline src/ingameWindows/iwShip.cpp]: Still a missing space

**Pull request merged**